src/digitKeyboard.js

```diff
@@ -0,0 +1,33 @@
+"use strict";
+
+import { EventDispatcher } from './eventDispatcher.js';
+
+/**
+ *  DigitKeyboard sets the digits on the screen.
+ */
+export class DigitKeyboard extends EventDispatcher {
+
+    /**
+     *  Constructor.  
+     */
+    constructor() {
+        super();
+        this._bindEvents();
+    }
+
+    /**
+     * Triggers binded event with a given value.
+     */
+    _bindEvents() {
+        let buttons = document.querySelector('.allButtons');
+        let numberButtons = buttons.querySelectorAll('.number');
+
+        for (let i = 0; i < numberButtons.length; i++) {
+            let self = this;
+            numberButtons[i].addEventListener("click", function() {
+                let buttonDigit = this.innerHTML;
```

Comment:
This is a not good practice, to obtain the innerHTML value this way. When using `this` inside of a class it should always reference to its instance. In this case it is referencing to a button.

The same way you are triggering some event with a value, `click` is triggered with a `MouseEvent`, which you can use retrieve all data about click - element which triggered it, phase in which it is, etc. Having that in mind, improve your handler to read data from the event.